src/Styles/Btn.css

```diff
@@ -0,0 +1,3 @@
+.btn-container {
```

Comment:
You should never apply a global css rule like that.

What would happen if you dont want the 30px?
Create more components?
start writing multiple rules?